synapse/cryotank.py

```diff
@@ -1256,18 +1100,18 @@ def _iterrows(self, prop, valu, exact=False):
                 if not curs.next():
                     return
 
-    def queryNormValu(self, prop, valu=None, exact=False):
+    def queryNormValu(self, prop: str, valu: Optional[Union[int, str]] = None, exact=False):
```

Comment:
If we're adding type annotations to these methods, `exact` should get one too

Reply:
typing is smart enough to infer the type from the constant.